### ISSUE TITLE
Region-wide search

### ISSUE THREAD

**Issue by sascha11110** [digitalfabrik/integreat-cms#62]
A page-wide search would make navigation much easier. In order to ensure a good user experience, this search should deliver the results _live_.

**Comment by svenseeberg** [comment]
Just to be sure: We are talking about a search in the the backend here and not providing a search API for apps? What should be searched? Menu items or the content of all pages? In all languages? I'm not yet sure right now what the most helpful solution is.

**Comment by sascha11110** [comment]
Yeah, a search for the backend. All entities should be searchable and I think it also makes sense to allow a search in all languages. To provide really good and fast search results, a normal SQL search will probably not be enough. Maybe it makes sense to set up elasticsearch or something similar. But this is also a discussion for the future ;)

**Comment by ulliholtgrave** [comment]
@svenseeberg I suggest to move the issue to the Version 1.0 milestone as search fields are important feature that we have in wordpress as well. Your opinion?

**Comment by timoludwig** [comment]
I think this issue covers the global search in one region which should return arbitrary results (so no matter what content type)... this doesn't exist in WordPress in this way, right? Thus I think we only need the object-based search #63 and can implement this one later on...

**Comment by JoeyStk** [comment]
@hauf-toni / @osmers What do you think about this issue? Is this needed? :)

**Comment by JoeyStk** [comment]
@laravoelk do you think I would be helpful for your team to be able to search region-wide? If not, I would close this issue :)
